skype/skype-ps/skype/Grant-CsTeamsAudioConferencingPolicy.md

```diff
@@ -0,0 +1,152 @@
+---
+external help file: MicrosoftTeams-help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/en-us/powershell/module/skype/grant-csteamsaudioconferencingpolicy
+schema: 2.0.0
+---
+
+# Grant-CsTeamsAudioConferencingPolicy
+
+## SYNOPSIS
+
+Assigns a Teams audio-conferencing policy at the per-user scope. Audio conferencing policies are used to manage audio conferencing toll- and toll-free numbers to be displayed in meeting invites created by users within your organization.
+
+## SYNTAX
+
+### GrantToTenant (Default)
+```
+Grant-CsTeamsAudioConferencingPolicy [-Global] [-PassThru] [[-PolicyName] <String>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### Identity
+```
+Grant-CsTeamsAudioConferencingPolicy [-PassThru] [[-PolicyName] <String>] [[-Identity] <String>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Granular control over which audio conferencing features your users can or cannot use is an important feature for many organizations. This cmdlet lets you assign a teams audio conferencing policy at the per-user scope. Audio conferencing policies determine the audio-conferencing toll- and toll-free numbers to be displayed in meeting invites created by users within your organization.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+PS C:\> Grant-CsTeamsAudioCOnferencingPolicy -identity "Ken Myer" -PolicyName “Emea Users”
+```
+
+In this example, a user with identity "Ken Myer" is being assigned the “Emea Users” policy
+
+
+
+## PARAMETERS
+
+### -Global
+
```

Comment:
Please add a description.

Reply:
This needs to be removed, it's not exposed